src/pages/en/guides/styling.md

```diff
@@ -81,11 +67,12 @@ import PostContent from './Post.astro';
 </article>
 ```
 
-This is a great way to style things like blog posts, or documents with CMS-powered content where the contents live outside of Astro. But be careful when styling children unconditionally, as it breaks component encapsulation. Components that appear different based on whether or not they have a certain parent component can become unwieldy quickly.
+This is a great way to style things like blog posts, or documents with CMS-powered content where the contents live outside of Astro. But be careful: components that appear different based on whether or not they have a certain parent component can become difficult to troubleshoot.
 
-#### Global styles within style tag
 
-If you’d like to use global styles but you don’t want to use a dedicated CSS file, there is a `<style global>` escape hatch:
+## Astro `<style global>`
+
+To apply styles globally, without the need for a .css file or stylesheet link, Astro allows you to use `<style global>` within any Astro component, page or layout to wrap style definitions. 
```

Comment:
What do you think of this? Reworked slightly as I found the idea of “wrapping style definitions” slightly confusing.

```suggestion
To apply styles globally, without the need for a separate `.css` file or external stylesheet link, Astro allows you to use the `global` attribute on style declarations within any Astro component, page or layout. 
```